src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/ListWithPhysicalAndLogicalLengthFW.java

```diff
@@ -0,0 +1,331 @@
+            int newLimit = limit() + FIELD_SIZE_FIELD1;
+            checkLimit(newLimit, maxLimit());
+            buffer().putInt(limit(), (int) (value & 0xFFFF_FFFFL));
+            fieldsMask |= 1 << FIELD_INDEX_FIELD1;
+            limit(newLimit);
+            return this;
+        }
+
+        private StringFW.Builder field2()
+        {
+            return field2RW.wrap(buffer(), limit(), maxLimit());
+        }
+
+        public Builder field2(
+            String value)
+        {
+            assert (fieldsMask & ~0x03) == 0 : "Field \"field2\" is already set or subsequent fields are already set";
+            assert (fieldsMask & 0x01) != 0 : "Prior required field \"field0\" is not set";
+            StringFW.Builder field2RW = field2();
+            field2RW.set(value, StandardCharsets.UTF_8);
+            fieldsMask |= 1 << FIELD_INDEX_FIELD2;
+            limit(field2RW.build().limit());
+            return this;
+        }
+
+        public Builder field2(
+            StringFW value)
+        {
+            assert (fieldsMask & ~0x03) == 0 : "Field \"field2\" is already set or subsequent fields are already set";
+            assert (fieldsMask & 0x01) != 0 : "Prior required field \"field0\" is not set";
+            StringFW.Builder field2RW = field2();
+            field2RW.set(value);
+            fieldsMask |= 1 << FIELD_INDEX_FIELD2;
+            limit(field2RW.build().limit());
+            return this;
+        }
+
+        public Builder field2(
+            DirectBuffer buffer,
+            int offset,
+            int length)
+        {
+            assert (fieldsMask & ~0x03) == 0 : "Field \"field2\" is already set or subsequent fields are already set";
+            assert (fieldsMask & 0x01) != 0 : "Prior required field \"field0\" is not set";
+            StringFW.Builder field2RW = field2();
+            field2RW.set(buffer, offset, length);
+            fieldsMask |= 1 << FIELD_INDEX_FIELD2;
+            limit(field2RW.build().limit());
+            return this;
+        }
+
+        @Override
+        public Builder wrap(
+            MutableDirectBuffer buffer,
+            int offset,
+            int maxLimit)
+        {
+            super.wrap(buffer, offset, maxLimit);
+            fieldsMask = 0;
+            int newLimit = limit() + FIELD_OFFSET_FIELD0;
+            checkLimit(newLimit, maxLimit());
+            limit(newLimit);
+            return this;
+        }
+
+        @Override
+        public ListWithPhysicalAndLogicalLengthFW build()
+        {
+            assert (fieldsMask & 0x01) != 0 : "Required field \"field0\" is not set";
+            buffer().putInt(offset() + PHYSICAL_LENGTH_OFFSET, (int) (limit() & 0xFFFF_FFFFL));
+            buffer().putInt(offset() + LOGICAL_LENGTH_OFFSET, (int) (Integer.bitCount(fieldsMask) & 0xFFFF_FFFFL));
+            buffer().putInt(offset() + BIT_MASK_OFFSET, (int) (fieldsMask & 0xFFFF_FFFFL));
+            fieldsMask = 0;
```

Comment:
No need to reset after build, just make sure it is reset in `wrap` and `tryWrap` instead.